include/oneapi/dpl/pstl/hetero/algorithm_ranges_impl_hetero.h

```diff
@@ -649,8 +730,10 @@ __pattern_reduce_by_segment(_ExecutionPolicy&& __exec, _Range1&& __keys, _Range2
 
     //reduce by segment
     oneapi::dpl::__par_backend_hetero::__parallel_for(
-        ::std::forward<_ExecutionPolicy>(__exec), unseq_backend::__brick_reduce_idx<_BinaryOperator>{__binary_op},
-        __result_end, experimental::ranges::views::all_read(__idx), ::std::forward<_Range2>(__values),
+        oneapi::dpl::__par_backend_hetero::make_wrapped_policy<__reduce1_wrapper>(
+            ::std::forward<_ExecutionPolicy>(__exec)),
+        unseq_backend::__brick_reduce_idx<_BinaryOperator>{__binary_op}, __result_end,
+        experimental::ranges::views::all_read(__idx), ::std::forward<_Range2>(__values),
         experimental::ranges::views::all_write(__tmp_out_values))
         .wait();
```

Comment:
Sometimes we use `oneapi::dpl` "prefix" for referring to a library name (e.g. in the lines 732,733), sometimes we do not (e.g. in the lines 735-737). This is somewhat strange. I thought all our classes (including ranges and bricks) are placed within `namespace oneapi::dpl`; why we skip it sometimes but not all times? Honestly, I think it is excessive when referring to any library name.

I'm not suggesting to immediately change it everywhere, or even in this patch. But coming to an agreement and making the use of library names consistent (in one big effort or with over-time improvements) seems reasonable to me.

Reply:
I've created an issue not to forget about the suggestion: #532